presto-product-tests/conf/docker/multinode/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   hadoop-master:
     hostname: hadoop-master
-    image: 'teradatalabs/cdh5-hive:2.6.0-cdh5.6.0-sshd-4e54a19'
+    image: 'teradatalabs/cdh5-hive:2'
```

Comment:
Although we decided that using such simple tags like `2` is fine, i still believe that it is not descriptive and error prone. It is easy to overwrite the image with such simple tag when two developers have changed something and they are not coordinated. And it is hard to understand what the image with some particular tag contains. But if you think that it is not a problem - i don't insist. We can keep it as is.